src/main/java/technology/tabula/CommandLineApp.java

```diff
@@ -156,11 +160,13 @@ private void extractFile(File pdfFile, Appendable outFile) throws ParseException
             while (pageIterator.hasNext()) {
                 Page page = pageIterator.next();
 
-                if (pageArea != null) {
-                    page = page.getArea(pageArea);
+                if (pageAreas != null) {
+                    for (Pair<Integer, Rectangle> areaPair : pageAreas) {
```

Comment:
I mean here. You have access to the `Page` objects, so you can do the conversion at this point.

Reply:
Agree... tried to explain the thinking behind why I put the calculations the Page.java as opposed to CommandLineApp.java file... Please do confirm once more before I send the updated code.

PS: I apologize for asking to reconfirm

Reply:
Explanation is under the other review comment.

Reply:
No apologies needed!

Reply:
Changes done and committed.